testing/scripts/rest_helpers.py

```
@@ -0,0 +1,236 @@
+            btn = page.locator(
+                f".swh-api-key-revoke-btn[data-swh-apikey-id='{kid}']"
+            )
+            if btn.count() == 0:
+                continue
+            btn.scroll_into_view_if_needed()
+            btn.click()
+            modal = page.locator("#swh-apikey-revoke-modal")
+            modal.wait_for(state="visible", timeout=3000)
+            page.click("#swh-apikey-revoke-confirm")
+            modal.wait_for(state="hidden", timeout=3000)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)